crates/flipperzero/src/gui/mod.rs

```diff
@@ -1,3 +1,230 @@
 //! GUI service.
 
 pub mod canvas;
+pub mod icon;
+pub mod icon_animation;
+pub mod view;
+pub mod view_port;
+pub mod xbm;
+
+use crate::{
+    gui::{
+        canvas::CanvasView,
+        view_port::{ViewPort, ViewPortCallbacks},
+    },
+    input::InputEvent,
+};
+use core::{
+    ffi::c_char,
+    fmt::Debug,
+    ops::{Deref, DerefMut},
+};
+use flipperzero_sys::{
+    self as sys, furi::UnsafeRecord, Canvas as SysCanvas, Gui as SysGui, GuiLayer as SysGuiLayer,
+};
+
+/// System Gui wrapper.
+pub struct Gui {
```

Comment:
This new struct should be mentioned in the changelog in the `Added` section. See prior changelog entries for the style.